src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/MinMaxTable.cs

```diff
@@ -23,5 +28,20 @@ public MinMaxTableFunction(bool isMin)
         {
             _delegationCapability = isMin ? DelegationCapability.Min : DelegationCapability.Max;
         }
+
+        public override bool CheckInvocation(TexlBinding binding, TexlNode[] args, DType[] argTypes, IErrorContainer errors, out DType returnType, out Dictionary<TexlNode, DType> nodeToCoercedTypeMap)
+        {
+            Contracts.AssertValue(args);
+            Contracts.AssertValue(argTypes);
+            Contracts.Assert(args.Length == argTypes.Length);
+            Contracts.AssertValue(errors);
+
+            var fValid = CheckInvocation(args, argTypes, errors, out returnType, out nodeToCoercedTypeMap);
```

Comment:
nit: make it clearer that you're calling `base.CheckInvocation`?

Reply:
The IDE doesn't like that idea. Also none of the other functions do that.